[v2.12] Move CIS Benchmark to Compliance (#1867)
rancher/rancher-docs: docs/reference-guides/rancher-cluster-tools.md

> Rancher can run a security scan to check whether Kubernetes is deployed according to security best practices as defined in the CIS Kubernetes Benchmark.

Reviewer: ```suggestion
Rancher can run a security scan to check whether Kubernetes is deployed according to security best practices as defined in most recognized Kubernetes Security Benchmarks, such as STIG.
```
